doc/getting-started/building-from-source.rst

```diff
@@ -179,12 +179,12 @@ the following to build the hypervisor, device model, and tools:
 
 You can also build ACRN with your customized scenario:
 
-* Build with your own scenario configuration on the ``nuc6cayh``, assuming the
+* Build with your own scenario configuration on the ``nuc11tnbi5``, assuming the
```

Comment:
we should only reference boards in the supported hardware document (this one's not there)

Reply:
We will update the TGL NUC platform later.